controllers/AuthController.js

```diff
@@ -43,14 +41,21 @@ export default class AuthController {
       isVerified: false,
       verificationToken,
       password: hashedPassword,
-      type: 'user',
+      type: 'user'
     };
 
-    const registeredUser = await User.create(user);
+    const registeredUser = await models.User.create(user);
     const token = Helper.createToken({
       id: registeredUser.id,
-      email,
```

Comment:
Hey, could you please add the trailing commas back here? thanks

Reply:
I will see to that.